openc3-init/plugins/packages/openc3-tool-common/src/components/widgets/Widget.js

```diff
@@ -80,6 +90,32 @@ export default {
     },
   },
   methods: {
+    verifyNumParams(keyword, min_num_params, max_num_params, usage = '') {
```

Comment:
Mostly duplicate of config-parser.js code but that relies on a bunch of variables set as it parses through the config file